src/main/app/src/app/shared/material-form-builder/form-components/documenten-lijst/documenten-lijst.component.html

```diff
@@ -26,7 +26,10 @@
                 </ng-container>
                 <ng-container matColumnDef="titel">
                     <th mat-header-cell *matHeaderCellDef> {{'titel' | translate}} </th>
-                    <td mat-cell *matCellDef="let document"> {{document.titel}} </td>
+                    <td mat-cell *matCellDef="let document">
+                        <zac-document-icon [document]="document"></zac-document-icon>
+                        {{document.titel}}
```

Comment:
Wellicht kun je de titel ook in het zac-document-icon opnemen. Dan hoef je deze niet overal nog los eronder te zetten.

Reply:
Die had ik er in eerste instantie bij staan, maar vanwege de layout keuzes die eerder zijn gemaakt voor het informatie-object-view component was dat niet handig - daar wordt de bestandsnaam getoond ipv de titel. Ik had er voor kunnen kiezen om configurabel te maken welke tekst er getoond wordt, of om de titel middels een boolean input te verbergen, maar dit vond ik netter aangezien het component dan maar voor een enkel doel dient.